internal/pkg/cli/deploy/errors.go

```diff
@@ -0,0 +1,15 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package deploy
+
+import "fmt"
+
+type errSvcWithNoAliasDeployingToEnvWithImportedCerts struct {
+	name    string
+	envName string
+}
+
+func (e *errSvcWithNoAliasDeployingToEnvWithImportedCerts) Error() string {
+	return fmt.Sprintf("cannot deploy service %s without custom domain to environment %s with certificate imported", e.name, e.envName)
```

Comment:
nit: should we say like this?
```suggestion
	return fmt.Sprintf("cannot deploy service %s without http.alias to environment %s with certificate imported", e.name, e.envName)
```